src/test/acceptance/features/email-address.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Enter email address
+  In order to apply for the HTBHF programme
+  As a potential claimant
+  I want to enter my email address
+
+  Background:
+    Given I have entered my details up to the email address page
+
+  Scenario: Enter in a valid email address
+    When I enter a valid email address
+    Then I am shown the check details page
+
+  Scenario: Do not enter in an email address
+    When I do not enter an email address
+    Then I am informed that the email address is in the wrong format
+
+  Scenario Outline: Fill email address with invalid format
```

Comment:
Can this match the other scenarios `Fill email address with invalid format` -> `Enter in an invalid email address`

also maybe remove `in` for the scenarios: `Enter in a valid email address` -> `Enter a valid email address`

Reply:
The Scenario outline isn't matched to steps, just the when, given and then statements. 
Where we use 'Enter' in a step will say 'Enter in', so stuck with that for consistency.